### PR TITLE
PAB-4299: Missing device in a group is now ignored

### PR DESCRIPTION
**Pull request by kailol-sag**
Shruti, please make sure that this release note is what you want. I checked the C8Y doc for groups and tried to use the corresponding wording. Looks like they talk about "subassets" now and no longer "child devices" ... ???

### CODE REVIEW

File: content/release-10-18+/streaming-analytics-10-18+-bundle/10.18+_0.md [hunk 1]
```diff
@@ -53,6 +53,11 @@ You can now create a selection list for an Analytics Builder template parameter.
 You can define selection lists for types such as string, float, source or destination, or geofence and you can also select a specific value to be the default value. The values that you define for a selection list are then available for selection when you create instances of the model.
 See also [Adding a selection list for a template parameter](https://cumulocity.com/docs/streaming-analytics/analytics-builder/#adding-a-selection-list-for-a-template-parameter) in the user documentation.
 
+#### Missing subassets in a group
```
Comment (shrutidogra-23) [inline content/release-10-18+/streaming-analytics-10-18+-bundle/10.18+_0.md:56]: This is also true for assets and not just device group. If a device/asset is missing from an asset hierarchy we no longer throw an error. Does saying "subasset of a group " cover the assets too?

File: content/release-10-18+/streaming-analytics-10-18+-bundle/10.18+_0.md [hunk 2]
```diff
@@ -53,6 +53,11 @@ You can now create a selection list for an Analytics Builder template parameter.
...
+
+Previously, an error was thrown for an active model if a subasset of a group no longer existed in the inventory, for example, because a device was deleted.
+As of this version, missing subassets in a group are ignored and an error is no longer thrown.
```
Comment (shrutidogra-23) [inline content/release-10-18+/streaming-analytics-10-18+-bundle/10.18+_0.md:59]: Should we explicityly mention that of the group or asset is empty as a result of device/asset deletion the error would still be raised?

File: content/release-10-18+/streaming-analytics-10-18+-bundle/10.18+_0.md [hunk 3]
```diff
@@ -53,6 +53,12 @@ You can now create a selection list for an Analytics Builder template parameter.
 You can define selection lists for types such as string, float, source or destination, or geofence and you can also select a specific value to be the default value. The values that you define for a selection list are then available for selection when you create instances of the model.
 See also [Adding a selection list for a template parameter](https://cumulocity.com/docs/streaming-analytics/analytics-builder/#adding-a-selection-list-for-a-template-parameter) in the user documentation.
 
+#### Missing subassets in a device group or asset hierarchy
+
+Previously, an error was thrown for an active model if a subasset of a device group or asset hierarchy no longer existed in the inventory, for example, because a device was deleted.
```
Comment (shrutidogra-23) [inline content/release-10-18+/streaming-analytics-10-18+-bundle/10.18+_0.md:58]: Sorry just another little clarification, does active model here means that's deployed? Then that's not true. The error was thrown on redeployment of the model.

Comment (shrutidogra-23) [reply content/release-10-18+/streaming-analytics-10-18+-bundle/10.18+_0.md:58]: "error was thrown on model redeployment..."

File: content/release-10-18+/streaming-analytics-10-18+-bundle/10.18+_0.md [hunk 4]
```diff
@@ -53,6 +53,12 @@ You can now create a selection list for an Analytics Builder template parameter.
 You can define selection lists for types such as string, float, source or destination, or geofence and you can also select a specific value to be the default value. The values that you define for a selection list are then available for selection when you create instances of the model.
 See also [Adding a selection list for a template parameter](https://cumulocity.com/docs/streaming-analytics/analytics-builder/#adding-a-selection-list-for-a-template-parameter) in the user documentation.
 
+#### Missing subassets in a device group or asset hierarchy
+
+Previously, when activating a model, an error was thrown if a subasset of a device group or asset hierarchy no longer existed in the inventory, for example, because a device was deleted.
```
Comment (shrutidogra-23) [inline content/release-10-18+/streaming-analytics-10-18+-bundle/10.18+_0.md:58]: Only if the model is configured to receive events from that device group/ asset hierarchy though. Is that implied here or should be mentioned ? I am not sure..

**Review by shrutidogra-23 — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**